camel/src/main/java/com/github/theprez/manzan/configuration/DestinationConfig.java

```diff
@@ -67,6 +68,12 @@ public synchronized Map<String, ManzanRoute> getRoutes(CamelContext context) {
                     final String topic = getRequiredString(name, "topic");
                     ret.put(name, new KafkaDestination(name, topic, format, getUriAndHeaderParameters(name, sectionObj, "topic")));
                     break;
+                case "google-pubsub":
+                    final String projectId = getRequiredString(name, "projectId");
+                    final String topicName = getRequiredString(name, "topicName");
+                    final String serviceAccountKey = getRequiredString(name, "serviceAccountKey");
+                    ret.put(name, new GooglePubSubDestination(context, name, format, projectId, topicName, serviceAccountKey, getUriAndHeaderParameters(name, sectionObj, "projectId", "topicName", "serviceAccountKey")));
```

Comment:
I think format should come right before getUriAndHeaderParameters to be consistent with the other methods